playbooks/demo_nim_vios_hc.yml

```diff
@@ -2,14 +2,21 @@
 - name: "VIOS health check information"
   gather_facts: yes
   hosts: nimserver
+  collections:
+  - ibm.power_aix
   
   vars:
   #VIOSes to do health check.
     targets_v: 
-      - "vios1"
-      - "vios2"
+      - "vios1,vios2"
```

Comment:
what is the difference between seperating it into bullet points i.e.
```
target_v:
  - vios1
  - vios2
  - vios3
```

and having it comma separated?

Reply:
Having it comma separated means it's a pair of dual vioses. Next line will be next pair or single.